docs/_posts/2017-06-14-jekyll-3-5-0-released.markdown

```diff
@@ -0,0 +1,38 @@
+---
+title: 'Jekyll turns 3.5, oh my!'
+date: 2017-06-15 17:32:32 -0400
+author: parkr
+version: 3.5.0
+categories: [release]
+---
+
+Good news! Nearly 400 commits later, Jekyll 3.5.0 has been released into
+the wild. Some new shiny things you might want to test out:
+
+- Jekyll now uses Liquid 4, the latest!
```

Comment:
The entire [commit history](https://github.com/Shopify/liquid/compare/3-0-stable...4-0-stable) between 3.0 and 4.0